Add template data (#18)
w3c/vc-di-ecdsa-test-suite: CHANGELOG.md

# w3c/vc-di-ecdsa-test-suite  ChangeLog

> ## 1.0.0

Reviewer: Are we going to include changelogs in all test suites moving forward? and where are we planning to publish them?
```suggestion
## 1.0.0 - 2023-11-dd
```

### Added

> - A new reporter option that outputs the json used to create the report.

Reviewer: ```suggestion
- Add a new reporter option that generates the JSON used to create the report.
```

### Changed

> - `@digitalbazaar/mocha-w3c-interop-reporter` is not version `^1.5.0`.

Reviewer: ```suggestion
- Use `@digitalbazaar/mocha-w3c-interop-reporter@1.5.0`.
```
